package.json

```diff
@@ -15,6 +15,7 @@
     "dotenv": "^8.2.0",
     "google-spreadsheet": "^3.1.15",
     "mapbox-gl": "^1.13.0",
+    "o1": "^0.2.6",
```

Comment:
what is this module o1 ?

Reply:
oh i guess i installed o1 insted of open layers ol by mistake while setting up earlier and it got commited.. would remove this and commit again

Reply:
@ivanoats now its removed.. thanks for reminding